src/gf-platformid

```diff
@@ -42,6 +42,10 @@ else
     coreos_gf_run_mount "${tmp_dest}" "blocksize:4096"
 fi
 
+if [[ "$arch" = "s390x" && "$platformid" = "qemu" ]]; then
```

Comment:
i would just pass in `openstack` instead of qemu here: https://github.com/coreos/coreos-assembler/blob/rhcos-4.2-multiarch/src/cmd-build#L248 for s390x instead of this - that way at least what you're passing in is what you set it to.

Reply:
The reason I chose not to implement it that was was because I don't know the underlying build process. If you can guarantee that that the build command will always be run on s390x hardware, than I could do a `uname -m` lookup for the arch info. Instead, I trusted the $arch variable (defined in https://github.com/coreos/coreos-assembler/blob/d18ca36933291742ab5d5593bed5093877b8edc2/src/cmdlib.sh#L42) to supply the correct arch information in the case of a cross-compiled build.

That is not to say that it doesn't already use `uname -m` to get the arch information, but I figured I should play it safe in case there is something I missed.

Reply:
`cmdlib.sh` i executed at the beginning of cmd-build, so   `arch` should be exported and available. You shouldn't need to define that again. The builds are also run on the corresponding hardware that the images are generated for, so shouldn't be an issue.

Reply:
Ah I missed that. :)

Great - see updated PR.